catalog/content/docs/src/main/resources/_contents/integrating-contents.adoc

```diff
@@ -93,7 +93,7 @@ The following illustration shows these interfaces and how they are packaged.
 ||   |PreCreate    | +-------------\| \-------------------/ | |CreateResponse| |ReadRequest| |ReadResponse| |UpdateRequest| | \------------------/|
 ||   |StoragePlugin| |PostUpdate   ||                       | |cFFC          | | cFFC      | | cFFC       | | cFFC        | |                     |
 ||   |cFFC         | |StoragePlugin||                       | \--------------/ \-----------/ \------------/ \-------------/ |                     | +-----------------------------------\
-||   \-------------/ |cFFC         ||                       | +--------------\ +-------------\ +--------------\             |                     | |{branding} Mime Types Support      |
+||   \-------------/ |cFFC         ||                       | +--------------\ +-------------\ +--------------\             |                     | |${branding} Mime Types Support      |
```

Comment:
Corner is open on rendered Doc
<img width="938" alt="screen shot 2016-03-01 at 11 35 04 am" src="https://cloud.githubusercontent.com/assets/11355332/13437494/ce01b882-dfa1-11e5-9395-225b805c9c62.png">